controllers/flowcollector_controller.go

```diff
@@ -67,6 +74,10 @@ func NewFlowCollectorReconciler(client client.Client, scheme *runtime.Scheme) *F
 //+kubebuilder:rbac:groups=security.openshift.io,resources=securitycontextconstraints,resourceNames=hostnetwork,verbs=use
 //+kubebuilder:rbac:groups=security.openshift.io,resources=securitycontextconstraints,verbs=list;create;update;watch
 //+kubebuilder:rbac:groups=apiregistration.k8s.io,resources=apiservices,verbs=list;get;watch
+//+kubebuilder:rbac:groups=operators.coreos.com,resources=operatorgroups;subscriptions,verbs=create;delete;patch;update;get;watch;list
+//+kubebuilder:rbac:groups=kafka.strimzi.io,resources=kafkas;kafkatopics;kafkausers,verbs=create;delete;patch;update;get;watch;list
+//+kubebuilder:rbac:groups=loki.grafana.com,resources=lokistacks;network,verbs=create;delete;patch;update;get;watch;list
+//+kubebuilder:rbac:groups=apiextensions.k8s.io,resources=customresourcedefinitions,verbs=list;get;watch
 
```

Comment:
More a general comment here, but we should try commenting the reason for each permission.

Reply:
Sure I can put some effort on this. Also I guess we can reduce some permissions :+1: